cms/djangoapps/contentstore/tasks.py

```diff
@@ -560,3 +561,16 @@ def get_dir_for_filename(directory, filename):
                 from .views.entrance_exam import add_entrance_exam_milestone
                 add_entrance_exam_milestone(course.id, entrance_exam_chapter)
                 LOGGER.info(u'Course %s Entrance exam imported', course.id)
+
+
+@task(name='contentstore.update_outline_from_modulestore_task')
```

Comment:
Good point. I got in the habit of the explicit names because of weird issues I saw when arguments change and we're between versions. But I should keep to the convention.